src/wx/panel.cpp

```diff
@@ -578,14 +580,14 @@ bool GameArea::LoadState()
 bool GameArea::LoadState(int slot)
 {
     wxString fname;
-    fname.Printf(SAVESLOT_FMT, game_name().mb_str(), slot);
+    fname.Printf(SAVESLOT_FMT, game_name().wc_str(), slot);
```

Comment:
What is your thinking about when we should use this `UTF8()` function as opposed to `.wc_str()` or `.c_str()`.

Do we want to aim for consistency in all the wx code with respect to this?

Reply:
We should always use `UTF8()` when we are going to call a core function for path, but it has to be as late as possible.
`.c_str()` is very destructive for our issue because it uses current locale encoding implicitly. We lose chars not recognized by the current codepage.
`.wc_str()` is always safe for us, although printing may be a problem (as in to status bar or drawing on screen).

We just must do this when calling our core functions. I don't think there are any major issues elsewhere.